.github/workflows/build-project.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Project
+on:
+  push:
+    branches:
+      - main
```

Comment:
It should be ok to have only "main" for now, until will stabilize the CI. But I'm thinking that in future we should add also releases ("release/yocto-*")